### PR TITLE
feat(query): support show columns query

### LINKED ISSUE #10698: Feature: show full columns from x.x
**Summary**
could support the grammar like :show full columns from x.x .  because some system like dataease (https://github.com/dataease/dataease) only support MYSQL ,not support databend now. 

if databend can support more MYSQL feature,i believe databend will become more popular.
the following is MYSQL:
0834ab6709cd109a8c7c7cf8ebc59bad

the following is DATABEND:
a6f452e672706c06c52288c61c9bd8c7

and the following is the error when use dataease:
a11b3c90c0fc9ba651f1f5b14d0846be

### PR DESCRIPTION
**Pull request by TCeason**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

support : "`SHOW [FULL] COLUMNS FROM <table> [FROM|IN <catalog>.<database>] [<show_limit>]`"

Closes #10698

### CODE REVIEW

**Comment by TCeason** [comment]
cc @soyeric128

File: src/query/ast/src/ast/statements/columns.rs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        write!(f, " COLUMNS FROM")?;
+
+        if let Some(database) = &self.database {
+            if let Some(catalog) = &self.catalog {
+                write!(f, "{catalog}.",)?;
+            }
+            write!(f, "{database}.",)?;
+        }
+
+        write!(f, " {}", self.table)?;
```
Comment (andylokandy) [inline src/query/ast/src/ast/statements/columns.rs:45]:
```suggestion
        write_period_separated_list(
            f,
            self.catalog
                .iter()
                .chain(&self.database)
                .chain(Some(&self.table)),
        )?;
```

File: src/query/ast/src/ast/statements/columns.rs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    pub limit: Option<ShowLimit>,
+}
+
+impl Display for ShowColumnsStmt {
+    fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
+        write!(f, "SHOW")?;
+        if self.full {
+            write!(f, " FULL")?;
+        }
+        write!(f, " COLUMNS FROM")?;
```
Comment (andylokandy) [inline src/query/ast/src/ast/statements/columns.rs:36]:
```suggestion
        write!(f, " COLUMNS FROM ")?;
```

**Review by andylokandy — APPROVED** [review]

**Comment by andylokandy** [comment]
Would you like to add a statement test in `src/query/ast/tests/it/parser.rs`?

**Review by b41sh — APPROVED** [review]

**Comment by soyeric128** [comment]
> cc @soyeric128

comment -> **C**omment

Image 2023-3-26 at 20 58

**Pull request merged**